demisto_sdk/commands/prepare_content/preparers/marketplace_incident_to_alert_playbooks_prepare.py

```diff
@@ -0,0 +1,127 @@
+import logging
+import re
+
+from demisto_sdk.commands.common.constants import MarketplaceVersions
+
+logger = logging.getLogger("demisto-sdk")
+
+
+class MarketplaceIncidentToAlertPlaybooksPreparer:
+    @staticmethod
+    def prepare(
+        data: dict,
+        marketplace: MarketplaceVersions = MarketplaceVersions.XSOAR,
+    ) -> dict:
+        """
+        Iterate over all the given content item descriptions and name fields and if a description or name field contains
+        the word incident / incidents, then replace it with alert / alerts in case of XSIAM Marketplace.
+        In any case (for all Marketplaces) remove wrapper (<-incident-> to incident, <-incidents-> to incidents).
+        Args:
+            data: content item data
+            marketplace: Marketplace.
+
+        Returns: A (possibly) modified content item data
+
+        """
+        # Replace incidents to alerts only for XSIAM
+        if marketplace == MarketplaceVersions.MarketplaceV2:
+            MarketplaceIncidentToAlertPlaybooksPreparer.prepare_descriptions_and_names(
+                data, True
+            )
+
+        # For all Marketplaces
+        MarketplaceIncidentToAlertPlaybooksPreparer.prepare_descriptions_and_names(
+            data, False
+        )
+
+        return data
+
+    @staticmethod
+    def prepare_descriptions_and_names(data: dict, replace_incident_to_alert: bool):
+
+        for k, v in data.get("tasks", {}).items():
+            if description := v.get("task", {}).get("description", ""):
+                if replace_incident_to_alert:
+                    MarketplaceIncidentToAlertPlaybooksPreparer.replace_incident_to_alert(
+                        data, "description", description, k
+                    )
+                else:
+                    MarketplaceIncidentToAlertPlaybooksPreparer.remove_wrapper_from_incident(
+                        data, "description", description, k
+                    )
```

Comment:
Can we turn these lines one function, and call it here?
They're repeated here, as well as in 53-59, 64-70 and 75-82